src/videotoolbox.cs

```diff
@@ -146,6 +150,24 @@ interface VTCompressionPropertyKey {
 		[Field ("kVTCompressionPropertyKey_ICCProfile")]
 		NSString ICCProfile { get; } 
 
+		// AlphaChannelMode
+		
+		[Mac (10,15), iOS (13,0), TV (13,0), Watch (6,0)]
+		[Field ("kVTCompressionPropertyKey_AlphaChannelMode")]
+		NSString AlphaChannelMode { get; } 
+		
+		[Mac (10,15), iOS (13,0), TV (13,0), Watch (6,0)]
+		[Field ("kVTAlphaChannelMode_StraightAlpha")]
+		NSString StraightAlpha { get; } 
+		
+		[Mac (10,15), iOS (13,0), TV (13,0), Watch (6,0)]
+		[Field ("kVTAlphaChannelMode_PremultipliedAlpha")]
+		NSString PremultipliedAlpha { get; } 
+		
+		[iOS (10,9)]
```

Comment:
Not sure about the iOS version here

Reply:
I think you also want `[Mac (10,9), iOS (13,0), TV (13,0), Watch (6,0)]` according to their docs https://developer.apple.com/documentation/videotoolbox/kvtcompressionpropertykey_gammalevel?language=objc

Reply:
Thanks boss 😄